mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageDAO.java

```diff
@@ -364,18 +364,18 @@ private PreparedStatement retrieveSelect(FetchType fetchType) {
     }
```

Comment:
I don't see the value of this commit b25eda71b868f92779fe9332b8eecd31eb2141b7. It removes 5 lines, but it introduces Mono in API where I don't think it's really necessary, no?

Reply:
Reverted